tests/unit/test_pull_request.py

```diff
@@ -115,13 +140,21 @@ def test_repository_client_create_branch(
     upstream_repository: Repo,
 ):
     """
-    arrange: given RepositoryClient and newly created files in repo directory
+    arrange: given RepositoryClient and newly created files in repo and repo/docs directory
```

Comment:
Should be `directories`

Reply:
in this case it is referring to the `repo/docs` directory rather than both in the `repo` and `docs` directory. Does that make sense? I'll add quotes around it to make it more obvious

Reply:
You are right, I misread which line you were referring to